node/src/accountant/test_utils.rs

```diff
@@ -1735,40 +1724,45 @@ impl ScanSchedulers {
     }
 }
 
-pub fn make_non_guaranteed_qualified_payable(n: u64) -> QualifiedPayableAccount {
-    // Without guarantee that the generated payable would cross the given thresholds
+pub fn make_meaningless_qualified_payable(n: u64) -> QualifiedPayableAccount {
+    // It's not guaranteed that the payables would cross the given thresholds.
+    let coefficient = (n as f64).sqrt().floor() as u64;
+    let permanent_deb_allowed_minor = gwei_to_wei(coefficient);
+    let payment_threshold_intercept = 7_u128 * gwei_to_wei::<u128, _>(n) / 10_u128;
     QualifiedPayableAccount::new(
         make_payable_account(n),
-        n as u128 * 12345,
-        CreditorThresholds::new(111_111_111),
+        payment_threshold_intercept,
+        CreditorThresholds::new(permanent_deb_allowed_minor),
     )
 }
 
-pub fn make_analyzed_account(n: u64) -> AnalyzedPayableAccount {
-    AnalyzedPayableAccount::new(make_non_guaranteed_qualified_payable(n), 123456789)
+pub fn make_meaningless_analyzed_account(n: u64) -> AnalyzedPayableAccount {
+    let qualified_account = make_meaningless_qualified_payable(n);
+    let disqualification_limit = 85 * qualified_account.payment_threshold_intercept_minor / 100;
+    AnalyzedPayableAccount::new(qualified_account, disqualification_limit)
 }
 
-pub fn make_guaranteed_qualified_payables(
+pub fn make_qualified_payables(
     payables: Vec<PayableAccount>,
     payment_thresholds: &PaymentThresholds,
     now: SystemTime,
 ) -> Vec<QualifiedPayableAccount> {
-    try_making_guaranteed_qualified_payables(payables, payment_thresholds, now, true)
+    try_to_make_guaranteed_qualified_payables(payables, payment_thresholds, now, true)
```

Comment:
Since, you have already decided to eliminate `guaranteed`, please remove it here too.